task_1.js

```diff
@@ -0,0 +1,113 @@
+var express = require('express');
+var app     = express();
+var http    = require('http')
+
+var PATH = '/I/want/title';
+
+app.get(PATH, function (req, res) {
```

Comment:
Its better to write `app.get`, `app.use` all together for any app. Create named functions and pass those. e.g. 

```js
app.get(PATH, processTitles);
app.use(notFoundHandler);
```

This makes more readable and maintainable code.